package.json

```diff
@@ -48,7 +48,7 @@
     "react-qrcode-logo": "^2.9.0",
     "react-redux": "^8.0.5",
     "react-router-dom": "^5.3.4",
-    "signify-ts": "github:WebOfTrust/signify-ts#faeebad4bdb29832cb25b83e41db064ede07dd36",
+    "signify-ts": "github:cardano-foundation/signify-ts#feat/ipexApplyOfferAgree",
```

Comment:
Can you put the specific githash of the latest commit on that branch instead of the branch name? In case there's another push, it's not deterministic.